src/tooltip/test/tooltip.spec.js

```diff
@@ -249,10 +249,11 @@ describe('tooltip', function() {
       scope.$digest();
     }));
 
-    it('should open after timeout', inject(function ($timeout) {
+    it('should open after timeout and set the close delay to the default', inject(function ($timeout) {
 
       elm.trigger('mouseenter');
       expect(tooltipScope.isOpen).toBe(false);
```

Comment:
The only way to "truly" test this would be to set up an interval or timeout and see if it respects the time it takes to close upon the trigger of the closing event of the popover. The reason I wrote the test this way was because of the test written for popup-delay. In order to provide consistent code, I wrote the test in the same manner. If there is a better way to test that actually works, then post your code please :D. Cheers!